assets/js/components/settings/SettingsActiveModules.js

```diff
@@ -84,28 +84,26 @@ const SettingsActiveModules = ( { activeModule, moduleState, setModuleState } )
 	return (
 		<Layout>
 			{ sortedModules.map( ( module ) => (
-				<SettingsModule
+				<SettingsActiveModule
 					key={ module.slug }
 					slug={ module.slug }
-					name={ module.name }
-					description={ module.description }
-					homepage={ module.homepage }
-					active={ module.active }
-					setupComplete={ module.active && module.connected }
-					onEdit={ onEdit }
-					onConfirm={ onConfirm }
-					onCancel={ onCancel }
-					isEditing={ { [ `${ activeModule }-module` ]: moduleState === 'edit' } }
+					onEdit={ onEdit.bind( null, module ) }
+					onConfirm={ onConfirm.bind( null, module ) }
+					onCancel={ onCancel.bind( null, module ) }
+					onToggle={ onToggle.bind( null, module ) }
```

Comment:
Using `bind` here undermines the use of `useCallback` above as a new function will be created every time.

Let's instead update the functions in `SettingsActiveModule` to provide the slug to the function provided here, or maybe move those handlers into that component instead.

Reply:
Updated.